shub/login.py

```diff
@@ -1,21 +1,56 @@
-import os, re, click
-from shub.utils import get_key_netrc, NETRC_FILE
+import re
+import click
+import os
+from shub import auth
+from scrapy.utils.conf import get_config
```

Comment:
talking about dependencies, this is making us dependant on Scrapy now for login.